config.py

```diff
@@ -1,13 +1,14 @@
 cfg = dict(
-    regexs = {
-"albums": r'href=\"\.\/albumarchive\/\d*?\/album\/(.*?)\" jsaction.*?>(?:<.*?>){5}(.*?)<\/div><.*?>(\d*?) ',
-"photos": r'\],\"(https:\/\/lh\d\.googleusercontent\.com\/.*?)\",\[\"\d{21}\"(?:.*?,){16}\"(.*?)\"',
-"review_loc_by_id": r'{}\",.*?\[\[null,null,(.*?),(.*?)\]'
-},
+    regexs={
```

Comment:
I think it reduces readability, are you sure it is in the pep-8 convention ?
It's seems weird, I'd rather not

Reply:
The dictionary is described in multiple lines, hence indentation facilitates reading. It's a quite standard way and it's much easier to see that `cfg` has all those kv.. Or are you talking about something else?

Reply:
@outime Yes the indentations are okay, I was talking about the "regexs={", isn't it better with whitespaces ?

Reply:
@mxrch ah gotcha! Let me show you an alternative way in a second.

Reply:
@mxrch here's one alternative https://github.com/mxrch/GHunt/pull/43/commits/cc2aa8c46ee033d56dfda8015cc9b185c0362689 if you don't like it then I can drop it and go back to the previous one with spaces.